Frameworks/UIKit/UIView.mm

```diff
@@ -409,6 +408,13 @@ - (UITouchPhase)_processPointerEvent:(WUXIPointerRoutedEventArgs*)pointerEventAr
             // Add this touch to the set of all current touches in the app
             [s_allTouches addObject:touchPoint.touch];
 
+            if (DEBUG_HIT_TESTING_LIGHT) {
+                TraceVerbose(TAG, L"Hit testing view %hs(0x%p).", object_getClassName(self), self);
+            }
+
+            // Update the hit-tested view. (Only one owner per UITouch lifetime)
+            touchPoint.touch->_view = [self _doHitTest:touchPoint.touch allTouches:s_allTouches];
```

Comment:
ah - i see you just moved it up from below.  maybe we can get rid of that call now altogether, or at least wrap it in the if (DEBUG_HIT_TESTING_LIGHT) block and get rid of the 'enable legacy hit-testing' path?

---
In reply to: [84356602](https://github.com/Microsoft/WinObjC/pull/1168#discussion_r84356602) [](ancestors = 84356602)